_includes/manuals/1.14/4.3.6_smartproxy_puppet.md

```diff
@@ -70,6 +74,16 @@ The [HOCON-formatted auth.conf style](https://docs.puppet.com/puppetserver/lates
 },
 {
     match-request: {
+    path: "/puppet/v3/environment_classes"
+       type: path
+       method: get
+      }
```

Comment:
Could you fix the indentation to match the other blocks here, and lines 82/83? (remove 3 leading spaces)